invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/subjects.html

```diff
@@ -6,13 +6,31 @@
   it under the terms of the MIT License; see LICENSE file for more details.
 #}
 
-{% set subjects = record.ui.subjects %}
+{% set subjects = record.metadata.subjects %}
 
 {% if subjects %}
-<dt><b>{{_('Subjects')}}</b></dt>
-<dd class="top-bottom-padded">
-  {%- for subject in subjects %}
-  <span class="label label-keyword label-aligned">{{ subject.title_l10n}}</span>
+<dd class="top-bottom-padded" id="record-subjects">
+  {%- for scheme, subjects in subjects|selectattr("scheme")|groupby('scheme') %}
+  <div class="ui grid">
+    <div class="sixteen wide subject-scheme column">
+      <span class="ui tiny subject header">{{scheme or _('Keywords')}}</span>
+    </div>
+    <div class="sixteen wide subject-label column">
+      {%- for subject in subjects %}
+          <span class="subject">{{ subject.subject}}</span>
+        {%- endfor %}
+    </div>
+  </div>
+  {%- endfor %}
+  {%- for subject in subjects|rejectattr("scheme")%}
```

Comment:
Probably just a `if subjects|rejectattr("scheme")` here and a `for subject in subjects|rejectattr("scheme")` below :smile:

Reply:
@zzacharo  Bumping this one, since as it is "Keywords" will be repeated for each subject without a scheme.